sky/global_user_state.py

```diff
@@ -39,6 +40,13 @@ def create_table(cursor, conn):
         last_use TEXT,
         status TEXT,
         autostop INTEGER DEFAULT -1)""")
+    # Table for Cluster History
+    cursor.execute("""\
+        CREATE TABLE IF NOT EXISTS cluster_history (
+        cluster_hash TEXT PRIMARY KEY,
+        name TEXT,
+        requested_resources TEXT,
+        metadata TEXT DEFAULT "{}")""")
```

Comment:
Since we only have `'usage_interval'` in the `metadata`, how about we just call it `usage_intervals`?

Reply:
I thought about this, but in the future it is possible we may have to add other keys to metadata. For that reason, I thought it was better to leave it is a dict since it will be more flexible.

Reply:
Could you talk a bit more about what we can possibly add to the metadata, and why we cannot add another column for that instead? IMO, the reason we use `metadata` in the cluster table is that clusters from the different backends can have some different metadata to be stored. As a database, it would be better to have the schema directly reflecting the content. Wdyt?

Reply:
Ah that is true, different columns would be clearer. Sure, I will change from dict to list.